cg/constants/constants.py

```diff
@@ -53,3 +53,5 @@ class DataDelivery(StrEnum):
 
 STATUS_OPTIONS = ("affected", "unaffected", "unknown")
 ANALYSIS_TYPES = ["tumor_wgs", "tumor_normal_wgs", "tumor_panel", "tumor_normal_panel"]
+
+SARS_COV_REGEX = "\(|\)|\d{2}CS\(|\)|\d{6}"
```

Comment:
I dont undestand why all these escaping characters are used. To match the string we want I would In bash do: "^[0-9]{2}CS[0-9]{6}$". Here "^" indicates beginning and "$" indicates end. Now sure how to translate that into python.